pkg/daemon/ceph/osd/daemon.go

```diff
@@ -84,7 +84,7 @@ func StartOSD(context *clusterd.Context, osdType, osdID, osdUUID, lvPath string,
 		logger.Errorf("failed to start osd or shutting down. %+v", err)
 	}
 
-	if pvcBackedOSD {
+	if pvcBackedOSD && !lvBackedPV {
```

Comment:
Out of curiosity why don't you need this?

Reply:
@satoru-takeuchi ?

Reply:
Oops, sorry. I overlooked this comment.

After running ceph-volume, OSD-on-PVC manages(activates/deactivates) VG since it is dedicated to OSD. However, it shouldn't be done in OSD-on-LV-backed-PVC case since VG is not only for OSD, but also for many PVCs. So CSI driver is in charge of managing VG. On the same reason, activating VG (the code block from line#60) also be skipped.

Is my answer makes sense?

Reply:
Makes sense! Thanks for the reply.